extensions/inference/inference_ops_cuda.py

```diff
@@ -12,12 +12,13 @@ def sources_files(self):
             for fname in [
                 "cuda/colossal_inference_C_frontend.cpp",
                 "cuda/decode_kv_cache_memcpy_kernel.cu",
+                "cuda/rms_layernorm_kernel.cu",
             ]
         ]
         return ret
 
     def include_dirs(self):
-        ret = [self.get_cuda_home_include()]
+        ret = [self.csrc_abs_path("cuda/include"), self.get_cuda_home_include()]
```

Comment:
Why is this modification made here?

Reply:
The rmsnorm cuda kernel depends on the block_reduce library under cuda/include, thus we need add the library path to the include directories when building the executable